README.md

```diff
@@ -306,50 +258,32 @@ datadog_agent::tags:
 
 These variables can be set in the `datadog_agent` class to control settings in the Agent:
 
-#### 2.x
-
 | variable name               | description                                                                                                                                                                                      |
 |-----------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `agent_major_version`       | The version of the Agent to install: either 5, 6 or 7 (default: 7).                                                                                                                              |
```

Comment:
I think `agent_repo_uri` should be mentioned here, as it's the only way to use a custom repo. For ubuntu / debian, `agent_default_release` should be documented as well. 

Moreover, the behavior and expected values of `agent_repo_uri` should be indicated (right now, you have to dig through the source code to know what you're supposed to do):
- for `apt`, the repo url (without release branch and component) (eg. `https://apt.datadoghq.com/`)
- for `yum`, the repo url with the release branch, component and arch (eg. `https://yum.datadoghq.com/stable/7/x86_64`)
- for Windows, the bucket url prefix (eg. `https://https://s3.amazonaws.com/ddagent-windows-stable/`)

Reply:
This was undocumented before A7 support, so I left as it was. I documented both in `init.pp` (in the source code, but it's on most parameter descriptions are).